packages/scripts/src/generateIndex.ts

```diff
@@ -61,7 +61,18 @@ export function tryResolveSummaryText(item: ApiDeclaredItem): string | null {
 				retVal += (node as DocPlainText).text;
 				break;
 			case DocNodeKind.LinkTag:
-				retVal += (node as DocLinkTag).urlDestination;
+				{
+					const { codeDestination, urlDestination, linkText } = node as DocLinkTag;
+					if (codeDestination) {
+						const declarationReference = item.getAssociatedModel()?.resolveDeclarationReference(codeDestination, item);
+						const foundItem = declarationReference?.resolvedApiItem;
+						retVal += urlDestination ?? linkText ?? foundItem?.displayName;
+						break;
+					}
+
+					retVal += linkText ?? urlDestination;
+				}
+
 				break;
```

Comment:
This looks rather weird IMO. Can you make the entire case body a block instead of having a hanging block within the case body?